.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  push:
```

Comment:
[minor] Probably this can be removed with some configurations as well as https://github.com/scalar-labs/scalardb/pull/1620